x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/AbstractArrayState.java

```diff
@@ -19,7 +19,7 @@
  *     Most of this class subclasses are autogenerated.
  * </p>
  */
-public class AbstractArrayState implements Releasable {
+public abstract class AbstractArrayState implements Releasable, GroupingAggregatorState {
```

Comment:
If I make this `abstract` and `implements` `GroupingAggregatorState` it's a little easier to poke it in tests.